sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -63,6 +63,7 @@
     'priority': 'priorityLevel',
     'no_response': 'responsePayloadOnWriteDisabled',
     'max_item_count': 'maxItemCount',
+    'throughput_bucket': 'throughputBucket'
```

Comment:
missing comma here is breaking the CI
```suggestion
    'throughput_bucket': 'throughputBucket',
```